tests/drivers/flash/common/boards/ek_ra4e2.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &storage_partition;
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* Set the 2 last block of storage. */
```

Comment:
@khoa-nguyen-18 : Is it mistake? If it is correct, please check similar point.

Reply:
I'm sure this information is correct with the hardware manual, I also checked with other boards. Thank you, can you please take a look?

Reply:
Confirmed that comment updated. Thanks @khoa-nguyen-18 for your update.